larwirecell/Components/DepoSetSimChannelSink.h

```diff
@@ -0,0 +1,72 @@
+/**
+ * Same as SimChannelSink but takes in DepoSet other than Depo
+ */
+
+#ifndef LARWIRECELL_COMPONENTS_DEPOSETSIMCHANNELSINK
+#define LARWIRECELL_COMPONENTS_DEPOSETSIMCHANNELSINK
+
+#include "WireCellIface/IAnodePlane.h"
+#include "WireCellIface/IConfigurable.h"
+#include "WireCellIface/IDepoSetFilter.h"
+#include "WireCellIface/IRandom.h"
+#include "WireCellUtil/Pimpos.h"
+#include "lardataobj/Simulation/SimChannel.h"
+#include "larwirecell/Interfaces/IArtEventVisitor.h"
+
+namespace wcls {
+
+  class DepoSetSimChannelSink : public IArtEventVisitor,
+                                public WireCell::IDepoSetFilter,
+                                public WireCell::IConfigurable {
+
+  public:
+    DepoSetSimChannelSink();
+    // virtual ~DepoSetSimChannelSink();
```

Comment:
Comments are fine, in general, but we suggest removing commented-out *code*, here and elsewhere.